packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './awards_section.css';
+
+export const AwardsSection = () => {
+    return (
+        <div className="awards-section-hack-aubg-container">
+            <h1 className="awards-section-title">AWARDS</h1>
+            <div className="awards-section-images-container">
+                <div className="awards-pic-text">
+                    <img
+                        src="awardIconSilver.png"
+                        alt="Award Silver"
+                        id="silver-img"
+                    ></img>
+                    <p className="award-money" id="silver-money-amt">
+                        2000 BGN
+                    </p>
+                </div>
+                <div className="awards-pic-text">
+                    <img
+                        src="awardIconGold.png"
+                        alt="Award Gold"
+                        id="gold-img"
+                    ></img>
+                    <p className="award-money" id="gold-money-amt">
+                        3000 BGN
+                    </p>
+                </div>
+                <div className="awards-pic-text">
+                    <img
+                        src="awardIconBronze.png"
+                        alt="Award Bronze"
+                        id="bronze-img"
+                    ></img>
+                    <p className="award-money" id="bronze-money-amt">
+                        1000 BGN
+                    </p>
+                </div>
+            </div>
+            <div className="awards-section-info-text">
+                <p id="awards-info-text">
+                    And much more!<br></br>
```

Comment:
Text is looking kind of weird, maybe enlarge it a bit and center it? Design peeps should share their opinion here - @annakukova @dankneebee. Furthermore, text size should be consistent with the text in other components whenever we change resolutions. 

`<br></br>` could be written as `<br/>`

Reply:
Reference: 
![image](https://user-images.githubusercontent.com/104720011/217642497-746ebb84-0f89-46de-856d-127f17c376a2.png)

Reply:
done